internal/subshell/tcsh/tcsh.go

```diff
@@ -48,17 +49,11 @@ func (v *SubShell) SetBinary(binary string) {
 }
 
 // WriteUserEnv - see subshell.SubShell
-func (v *SubShell) WriteUserEnv(cfg sscommon.Configurable, env map[string]string, envType sscommon.RcIdentification, _ bool, create bool) error {
+func (v *SubShell) WriteUserEnv(cfg sscommon.Configurable, env map[string]string, envType sscommon.RcIdentification, _ bool) error {
```

Comment:
Why does the bool argument still exist?

Reply:
That argument is `isAdmin` for the `cmd` shell. It was there originally.